src/openapi/streaming/connection/transport/websocket-transport.ts

```diff
@@ -283,10 +283,18 @@ class WebsocketTransport implements StreamingTransportInterface {
 
     private createSocket() {
         try {
-            const url = this.normalizeWebSocketUrl(
+            let url = this.normalizeWebSocketUrl(
                 `${this.connectionUrl}${this.query}`,
             );
 
+            if (this.isWebsocketStreamingHeartBeatEnabled) {
+                if (url.indexOf('?') === -1) {

```

Comment:
the query always contains the authorization - below there are assumptions we can just add `&`

Reply:
yes that can be done
updated